container/container.scss

```diff
@@ -0,0 +1,71 @@
+.container {
+  /* width: 590px;
+  margin: 0 auto;*/
+}
+
+
+
+.header__container {
```

Comment:
why is this located inside container? It relates to the header block - thus is should be  in the file of the header block